packages/semantic-tokens/scripts/tokenGen.ts

```diff
@@ -0,0 +1,298 @@
+/*
+ * Token generation script
+ * Takes in a Figma token export file and generates token raw strings and CSS Var files
+ */
+import tokensJSONRaw from './tokens.json';
+import { fluentOverrides as fluentFallbacksRaw, FluentOverrideValue, type FluentOverrides } from './fluentOverrides';
+import fs from 'node:fs';
+import { Project } from 'ts-morph';
+import { format } from 'prettier';
+import { ComponentTokenMap, Token } from './token.types';
+import path from 'node:path';
+import { dedupeShadowTokens } from '../utils/dedupeShadowTokens';
+import { toCamelCase } from '../utils/toCamelCase';
+import { escapeInlineToken } from '../utils/escapeInlineToken';
+import { removeLastDelimiter } from '../utils/removeLastDelimiter';
```

Comment:
create a barrel file in utils (utils/index.ts) and export them there as: 

```ts
export { chopLastCamelCasePart } from './chopLastCamelCasePart';
export { toCamelCase } from './toCamelCase';
export { removeLastDelimiter } from './removeLastDelimiter';
export { escapeInlineToken } from './escapeInlineToken';
export { dedupeShadowTokens } from './dedupeShadowTokens';
```

and later you can import like this: 

```ts
import { removeLastDelimiter, escapeInlineToken, toCamelCase, dedupeShadowTokens } from '../utils';
```